package.json

```diff
@@ -22,6 +22,7 @@
 		"ci:storybook": "cross-env NODE_ENV=development nx run storybook:build:ci --skip-nx-cache",
 		"clean": "yarn cleaner tag:component",
 		"clean:docs": "rimraf dist",
+		"clean:env": "rm -rf .env .storybook/chromatic.config.json",
```

Comment:
A more targeted clean-up than having to run `git clean -dfX` to remove the env files

Reply:
Nice. I did notice when I do run `git clean -dfX`, then a yarn install and yarn start, I get some errors with the chromatic config. Apparently with `git clean -dfX`, that `zip: "true"` should be a boolean, `zip: true`.
<img width="724" alt="Screenshot 2025-01-02 at 4 53 18 PM" src="https://github.com/user-attachments/assets/dc8918a4-9205-4c0d-9044-e9d15d5c7eb3" />
<img width="841" alt="Screenshot 2025-01-02 at 4 53 01 PM" src="https://github.com/user-attachments/assets/b81ce18c-5d06-40f8-aa41-579e56843ca9" />

If I manually change `"true"` to a boolean, rerun `yarn start`, I get not problems and the project runs. If I run `yarn clean:env` then `yarn start`, I get no problems and it runs. Would you say that it's not recommended to run `git clean -dfX` anymore? Why is that?

Reply:
Great call-out!  I pushed up a fix to the original bash script that creates the config to leave off the quotation marks when a boolean value is present.

Reply:
Awesome 👏